src/mobile/src/ui/views/wallet/AddCustomNode.js

```diff
@@ -193,7 +195,16 @@ class AddCustomNode extends Component {
             return this.onAddHttpNodeError();
         }
 
-        if (!nodes.includes(customNode.replace(/ /g, ''))) {
+        const hasDefaultHttpsPort = endsWith(customNode, ':443');
+
+        if (hasDefaultHttpsPort) {
+            customNode = customNode.slice(0, -4);
+        }
+
+        if (
+            !nodes.includes(customNode) &&
+            !map(nodes, (node) => (endsWith(node, ':443') ? node.slice(0, -4) : node)).includes(customNode)
```

Comment:
Why do we use `lodash.map` here but `Array.prototype.map` at https://github.com/iotaledger/trinity-wallet/pull/849/files#diff-14d3a021ba31141b14312a6343d8a557R65?

Reply:
No specific reason. Both are iterating over arrays here. 

However, https://github.com/iotaledger/trinity-wallet/pull/853 will remove this duplication.

Reply:
Oh ok. I was just wondering if there was a special reason